test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -399,26 +399,25 @@ def verify_last_message_sent_is_not(self, message: str):
     # This method expects to have just one mention / link in the last chat message 
     def verify_last_message_sent_contains_mention(self, displayName: str, message: str):
         [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
+
+        if not loaded:
+            verify_failure("No messages found in chat.")
         
-        if loaded:
-            # Verifying mention
-            verify_text_contains(str(last_message_obj.text), displayName)
-            
-            # Verifying message
-            verify_text_contains(str(last_message_obj.text), message)
-            
-            # Get link value from chat text:
-            try:
-                href_info = re.search(_LINK_HREF_REGEX, str(last_message_obj.text)).group(1)
-            except AttributeError:
-                # <a href=, "> not found in the original string
-                verify_failure("Mention link not found in last chat message.")
-            
-            click_link(ChatComponents.LAST_MESSAGE_TEXT.value, href_info)
-            verify(is_found(ChatComponents.MENTION_PROFILE_VIEW.value), "Checking user mentioned profile popup is open.")            
-            
-        else:
-            verify_failure("No messages found in chat.")  
+        # Verifying mention
+        verify_text_contains(str(last_message_obj.text), displayName)
+        
+        # Verifying message
+        verify_text_contains(str(last_message_obj.text), message)
+        
+        # Get link value from chat text:
+        try:
+            href_info = re.search(_LINK_HREF_REGEX, str(last_message_obj.text)).group(1)
+        except AttributeError:
+            # <a href=, "> not found in the original string
+            verify_failure("Mention link not found in last chat message.")
+        
+        click_link(ChatComponents.LAST_MESSAGE_TEXT.value, href_info)
+        verify(is_found(ChatComponents.MENTION_PROFILE_VIEW.value), "Checking user mentioned profile popup is open.")            
```

Comment:
sounds like a separate step (current fn name `verify_last_message_sent_contains_mention`)

Reply:
I checks the mention by opening the profile by clicking the link, so it kinda makes sense.

Anyway, that test is broken and will need to be fixed in another PR (issue here https://github.com/status-im/status-desktop/issues/9314)

Reply:
I mean, when I read the code and see fn `verify_last_message_sent_contains_mention` I'm expecting, that we are checking a mention in the msg, but we additionally do `verify(is_found(ChatComponents.MENTION_PROFILE_VIEW.value), "Checking user mentioned profile popup is open.")`, that means that you click on the mention and open a profile and this is 2 different steps:
- verify msg contains the mention
- verify mention click opens profile view

P.S. But it is not supercritical